client/src/components/home/HomeComponent.tsx

```diff
@@ -53,46 +53,38 @@ const callCreateRoom = async (role: RoomParticipantRole, props: HomeComponentPro
 export const HomeComponent = (props: HomeComponentProps): JSX.Element => {
   return (
     <Stack styles={fullScreenStyles}>
-      <Stack
-        horizontalAlign="center"
-        verticalAlign="start"
-        styles={containerStyles(props.theme)}
-        tokens={{ childrenGap: 15 }}
-      >
+      <Stack horizontalAlign="center" verticalAlign="start" tokens={{ childrenGap: 15 }}>
         <Stack styles={innerContainer}>
           <Stack verticalAlign="center" horizontalAlign="center">
             <Image imageFit={ImageFit.contain} src={imageCalendar} alt="calendarImage"></Image>
           </Stack>
           <Stack styles={containerMarginTop2rem}>
             <Text styles={lineHeight28px}>Hello,</Text>
             <Text styles={lineHeight22px}>What would you like to do?</Text>
-            <Stack horizontal styles={btnStackStyles} wrap horizontalAlign="space-evenly">
-              <StackItem>
-                <DefaultButton
-                  text="Book an appointment"
-                  styles={buttonStyles}
-                  iconProps={calendarIconStyles(props.theme)}
-                  onClick={() => window.location.assign('/book')}
-                />
-              </StackItem>
-              <StackItem>
-                <DefaultButton
-                  text="Start a call"
-                  splitButtonAriaLabel="See 2 options"
-                  styles={buttonStyles}
-                  iconProps={videoIconStyles(props.theme)}
-                  aria-roledescription="split button"
-                  menuProps={menuProps(props)}
-                />
-              </StackItem>
-              <StackItem>
-                <DefaultButton
-                  text="Join from link"
-                  styles={buttonStyles}
-                  iconProps={linkIconStyles(props.theme)}
-                  onClick={() => window.location.assign('/visit')}
-                />
-              </StackItem>
+            <Stack horizontal styles={btnStackStyles} wrap horizontalAlign="space-between">
+              <DefaultButton
+                styles={buttonStyles}
+                iconProps={calendarIconStyles(props.theme)}
+                onClick={() => window.location.assign('/book')}
+              >
+                <Text styles={buttonTextStyles}>Book an appointment</Text>
+              </DefaultButton>
+              <DefaultButton
+                splitButtonAriaLabel="See 2 options"
```

Comment:
should this be changed as we only have one option now

Reply:
That's my preference as well but the decision by the team was to keep it